roles/get_image_hash/tasks/main.yml

```diff
@@ -67,25 +67,26 @@
 - name: Update release_images with hashes
   when: get_release_images | bool
   block:
-    - name: Update released items
-      vars: 
+    - name: Update released items (image:tag format)
+      vars:
         updated_release_images: []
```

Comment:
I think we need to pre-populate `updated_release_images` with any fully qualified images, otherwise they will be dropped by the loop.

Reply:
+1

Reply:
pushed another commit, lmk if that is what you had in mind